### DISCUSSION TITLE
【Request feature】 request scheduler can set RestartOnFailure , please.

### DISCUSSION THREAD

**Discussion by we684123** [mhammond/pywin32#2533] [General]
* Expected behavior and actual behavior.

i follow [taskschedulerschema-restartonfailure](https://docs.microsoft.com/en-us/windows/win32/taskschd/taskschedulerschema-restartonfailure-settingstype-element) and [interval](https://docs.microsoft.com/en-us/windows/win32/taskschd/taskschedulerschema-interval-restarttype-element), [count](https://docs.microsoft.com/en-us/windows/win32/taskschd/taskschedulerschema-count-restarttype-element)

but i can't set task scheduler "RestartOnFailure" property
it return `AttributeError: <unknown>.RestartOnFailure`

* Steps to reproduce the problem.

```python
import datetime
import win32com.client

scheduler = win32com.client.Dispatch('Schedule.Service')
scheduler.Connect()
root_folder = scheduler.GetFolder('\\')
task_def = scheduler.NewTask(0)

# Create trigger
start_time = datetime.datetime.now() + datetime.timedelta(minutes=5)
TASK_TRIGGER_TIME = 1
trigger = task_def.Triggers.Create(TASK_TRIGGER_TIME)
trigger.StartBoundary = start_time.isoformat()

# Create action
TASK_ACTION_EXEC = 0
action = task_def.Actions.Create(TASK_ACTION_EXEC)
action.ID = 'DO NOTHING'
action.Path = 'cmd.exe'
action.Arguments = '/c "exit"'

# Set parameters
task_def.RegistrationInfo.Description = 'Test Task'
task_def.Settings.Enabled = True
task_def.Settings.StopIfGoingOnBatteries = False

# 🔴 can not set RestartOnFailure 🔴
task_def.Settings.RestartOnFailure.Count = 4
task_def.Settings.RestartOnFailure.Interval = "PT1M"

# Register task
# If task already exists, it will be updated
TASK_CREATE_OR_UPDATE = 6
TASK_LOGON_NONE = 0
root_folder.RegisterTaskDefinition(
    'Test Task',  # Task name
    task_def,
    TASK_CREATE_OR_UPDATE,
    '',  # No user
    '',  # No password
    TASK_LOGON_NONE)
```

* Version of Python and pywin32
python v3.8.5
pywin32 v228

Thanks for watching

**Comment by mhammond** [comment]
This sounds more like a support request than a bug report.

**Comment by we684123** [comment]
😅 
Sorry, I thought pywin32 had support for the entire task scheduler
so I thought it was a bug
I'll change the issue title

**Comment by mhammond** [comment]
Please try https://github.com/mhammond/pywin32#support

**Comment by AkshayKnorr** [comment]
hey I wanna access start task only if the computer is on AC how to access this option through this program
